server/channels/store/sqlstore/channel_store.go

```diff
@@ -124,6 +152,30 @@ func channelMemberToSlice(member *model.ChannelMember) []any {
 	return resultSlice
 }
 
+func channelToSlice(channel *model.Channel) []interface{} {
+	resultSlice := []interface{}{}
```

Comment:
Any reason not doing a `resultSlice := []any{channel.Id, channel.CreateAt, ...}`? Why separate appends for each field?

Reply:
Done now, makes sense.